src/Console/Command/CommandHelp.php

```diff
@@ -113,7 +113,7 @@ public static function getHelpCopy()
 
 Instead of using command line options to customize the rule, you can save the
 project configuration in a <comment>.php_cs.dist</comment> file in the root directory
-of your project. The file must return an instance of ``PhpCsFixer\ConfigInterface``,
+of your project. The file must return an instance of `PhpCsFixer\\\\ConfigInterface <https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.1/src/ConfigInterface.php>`_,
```

Comment:
No, as master points to 2.3.[...] and that is not on GH.
We only change the interface on a major anyway and than we can adjust this